server/parser.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	vitess "github.com/dolthub/vitess/go/vt/sqlparser"
+
+	"github.com/dolthub/doltgresql/postgres/parser/parser"
+	"github.com/dolthub/doltgresql/server/ast"
+)
+
+var _ sql.Parser = &PostgresParser{}
+
+type PostgresParser struct{}
```

Comment:
Should have comments on everything (structs and functions), explaining what it is, what it's for, etc.